src/main/java/com/team4/testingsystem/repositories/TestsRepository.java

```diff
@@ -69,12 +70,12 @@
 
     @Transactional
     @Modifying
-    @Query("update Test t set t.status = ?2 where t.id = ?1 and t.isAvailable = true")
+    @Query(value = "update Test t set t.status = ?2 where t.id = ?1 and t.isAvailable = true")
     int updateStatusByTestId(Long testId, Status newStatus);
 
     @Transactional
     @Modifying
-    @Query(value = "UPDATE Test t SET t.startedAt = ?1, t.status = 'STARTED' "
+    @Query("UPDATE Test t SET t.startedAt = ?1 "
```

Comment:
It's pretty strange that you added lots of default `value` parameters above, and delete on the single query